### PR TITLE
[Outlook] (preview) Add EnhancedLocation samples

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[59ef211](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/59ef211e1b68c1d59a07d15efb184c71c3fa2b6e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[59ef211](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/59ef211e1b68c1d59a07d15efb184c71c3fa2b6e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[59ef211](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/59ef211e1b68c1d59a07d15efb184c71c3fa2b6e)_: 

:clock10: **Incremental building**: average incremental building time is _5 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[59ef211](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/59ef211e1b68c1d59a07d15efb184c71c3fa2b6e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.enhancedlocation.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.enhancedlocation.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C31%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201901312014223920-307%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=khefi%2Fb5CUYok1tMBA71eJMNhHzBjQ74p3w3%2BvOZfGw%3D&st=2019-01-31T20%3A15%3A28Z&se=2019-03-03T20%3A20%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[20c9e6d](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/20c9e6d1e9dfacba5bdf30d5cca432e6da923a5c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[20c9e6d](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/20c9e6d1e9dfacba5bdf30d5cca432e6da923a5c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[20c9e6d](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/20c9e6d1e9dfacba5bdf30d5cca432e6da923a5c)_: 

:clock10: **Incremental building**: average incremental building time is _5 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e25bd39](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/e25bd393af06f9b91e576ed871d9487920ce31df)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[20c9e6d](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/20c9e6d1e9dfacba5bdf30d5cca432e6da923a5c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.enhancedlocation.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.enhancedlocation.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_1/Outlook_1_1.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_1/Outlook_1_1.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_2/Outlook_1_2.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_2/Outlook_1_2.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_3/Outlook_1_3.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_3/Outlook_1_3.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_4/Outlook_1_4.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_4/Outlook_1_4.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_5/Outlook_1_5.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_5/Outlook_1_5.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_6/Outlook_1_6.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_6/Outlook_1_6.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_7/Outlook_1_7.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_7/Outlook_1_7.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C31%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201901312034310485-307%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ni4QBXHRcux5Nkwj4Avdf9tpVEinVc58jKXjYHyTddY%3D&st=2019-01-31T20%3A35%3A37Z&se=2019-03-03T20%3A40%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e25bd39](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/e25bd393af06f9b91e576ed871d9487920ce31df)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e25bd39](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/e25bd393af06f9b91e576ed871d9487920ce31df)_: 

:clock10: **Incremental building**: average incremental building time is _5 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e25bd39](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/e25bd393af06f9b91e576ed871d9487920ce31df)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.enhancedlocation.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.enhancedlocation.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_3/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_4/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_5/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_6/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_7/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_1/Outlook_1_1.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_1/Outlook_1_1.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_2/Outlook_1_2.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_2/Outlook_1_2.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_3/Outlook_1_3.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_3/Outlook_1_3.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_4/Outlook_1_4.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_4/Outlook_1_4.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_5/Outlook_1_5.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_5/Outlook_1_5.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_6/Outlook_1_6.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_6/Outlook_1_6.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_7/Outlook_1_7.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_7/Outlook_1_7.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_1.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_1.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_2.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_2.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_3.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_3.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_4.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_4.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_5.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_5.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_6.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_6.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_7.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_7.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C31%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201901312040138086-307%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3csHJsWS4BbIVa01FzkNlm5jxila%2BAmgSVQPR9HDZ34%3D&st=2019-01-31T20%3A41%3A58Z&se=2019-03-03T20%3A46%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/docs-ref-autogen/outlook/office.enhancedlocation.yml [hunk 1]
```diff
@@ -87,6 +103,26 @@ items:
...
+
+      function callbackFunction(asyncResult) {
+          asyncResult.value.forEach(function (place) {
+                  console.log("Display name: " + place.displayName);
+                  console.log("Type: " + place.locationIdentifier.type);
+                  if (place.locationIdentifier.type == Office.MailboxEnums.LocationType.Room) {
+                      console.log("Email address: " + place.emailAddress);
+              }
+          });
+      }
```
Comment (kbrandl) [inline docs/docs-ref-autogen/outlook/office.enhancedlocation.yml]: Need to fix the indentation in this code sample (in the snippet file where this code sample originates from) -- the first several lines within the `forEach` are indented 4 spaces too far.

File: docs/docs-ref-autogen/outlook/office.appointmentread.yml [hunk 2]
```diff
@@ -555,6 +555,26 @@ items:
...
+
+      function callbackFunction(asyncResult) {
+          asyncResult.value.forEach(function (place) {
+                  console.log("Display name: " + place.displayName);
+                  console.log("Type: " + place.locationIdentifier.type);
+                  if (place.locationIdentifier.type == Office.MailboxEnums.LocationType.Room) {
+                      console.log("Email address: " + place.emailAddress);
+              }
+          });
+      }
```
Comment (kbrandl) [inline docs/docs-ref-autogen/outlook/office.appointmentread.yml]: Need to fix the indentation in this code sample (in the snippet file where this code sample originates from) -- the first several lines within the forEach are indented 4 spaces too far.

File: docs/docs-ref-autogen/outlook/office.appointmentcompose.yml [hunk 3]
```diff
@@ -595,6 +595,48 @@ items:
...
+      var locationDetails = [ 
+          {
+              "id": "Contoso",
+              "type": Office.MailboxEnums.LocationType.Custom
+          }
+      ];
+
+      Office.context.mailbox.item.enhancedLocation.addAsync(locationDetails);
+
+      Office.context.mailbox.item.enhancedLocation.getAsync(callbackFunction);
```
Comment (kbrandl) [inline docs/docs-ref-autogen/outlook/office.appointmentcompose.yml]: For some reason there's no blank line rendering in the published doc between the `addAsync` call and the `getAsync` call. Can you please add an extra blank line between these lines (in the snippet file, where this code originates)?

File: docs/docs-ref-autogen/outlook/office.appointmentcompose.yml [hunk 4]
```diff
@@ -595,6 +595,48 @@ items:
...
+
+      function callbackFunction(asyncResult) {
+          asyncResult.value.forEach(function (place) {
+                  console.log("Display name: " + place.displayName);
+                  console.log("Type: " + place.locationIdentifier.type);
+                  if (place.locationIdentifier.type == Office.MailboxEnums.LocationType.Room) {
+                      console.log("Email address: " + place.emailAddress);
+              }
+          });
+      }
```
Comment (kbrandl) [inline docs/docs-ref-autogen/outlook/office.appointmentcompose.yml]: Need to fix the indentation in this code sample (in the snippet file where this code sample originates from) -- the first several lines within the forEach are indented 4 spaces too far.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[3b0dae3](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/3b0dae3fbc7b9d84570d27b7de2e0ff422bdfc9c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[3b0dae3](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/3b0dae3fbc7b9d84570d27b7de2e0ff422bdfc9c)_: 

:clock10: **Incremental building**: average incremental building time is _5 min(s) 2 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[3b0dae3](https://github.com/OfficeDev/office-js-docs-reference/pull/307/commits/3b0dae3fbc7b9d84570d27b7de2e0ff422bdfc9c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.enhancedlocation.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook/office.enhancedlocation.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_2/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_3/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_3/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_4/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_5/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_6/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_6/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_7/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/docs/docs-ref-autogen/outlook_1_7/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_1/Outlook_1_1.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_1/Outlook_1_1.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_2/Outlook_1_2.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_2/Outlook_1_2.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_3/Outlook_1_3.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_3/Outlook_1_3.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_4/Outlook_1_4.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_4/Outlook_1_4.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_5/Outlook_1_5.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_5/Outlook_1_5.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_6/Outlook_1_6.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_6/Outlook_1_6.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_7/Outlook_1_7.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/api-extractor-inputs-outlook-legacy/Outlook_1_7/Outlook_1_7.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_1.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_1.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_2.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_2.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_3.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_3.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_4.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_4.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_5.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_5.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_6.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_6.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/Outlook_1_7.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/Outlook_1_7.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-add-enhancedLocation-samples/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C1%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201902011753126256-307%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=aJqDlc34buxSo1lNteO4v33o1LFLFfZSZ4IyeWQLfBg%3D&st=2019-02-01T17%3A54%3A12Z&se=2019-03-04T17%3A59%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**